lib/doro/behaviors/behavior.ex

```diff
@@ -18,6 +18,16 @@ defmodule Doro.Behavior do
     |> Enum.filter(&Regex.match?(~r/^Elixir\.Doro\.Behaviors\./, Atom.to_string(&1)))
   end
 
+  def find(nil), do: nil
+  def find(""), do: nil
+
+  def find(behavior) when is_binary(behavior) do
+    all_behaviors()
+    |> Enum.find(fn entry ->
+      entry == String.to_atom("Elixir.Doro.Behaviors.#{Macro.camelize(behavior)}")
+    end)
```

Comment:
You could stick this as the inverse of `Modules.to_underscore`, but it wouldn't be generalized to all modules I guess.  Or we could do `Behavior.to_string(behavior)` and `Behavior.from_string(s)` which would do these transforms.  It might make this stuff easier to read and centralize these mappings.